test/drafts/ERC1820.test.js

```diff
@@ -0,0 +1,51 @@
+const { shouldFail, singletons } = require('openzeppelin-test-helpers');
+const { bufferToHex, keccak256 } = require('ethereumjs-util');
+
+const ERC1820Mock = artifacts.require('ERC1820Mock');
+
+contract('ERC1820', function ([_, registryFunder, implementee, anyone]) {
+  beforeEach(async function () {
+    this.erc1820 = await ERC1820Mock.new();
```

Comment:
Likewise here, I'd add the word `implementer` to these variable names. Perhaps `this.implementer`, and `ERC1820ImplementerMock`.